modules/ROOT/pages/download-and-install-service-mesh.adoc

```diff
@@ -46,4 +46,42 @@ Anypoint Service Mesh is installed and a status of all pods is displayed:
 +
 image::verified-service-mesh-installer-output.png[]
 
+=== Install With Proxy
+
+You might need to install Anypoint Service Mesh in a proxy environment. In such cases, you must provide provide the proxy configuration during the installation.
+
+To install Anypoint Service Mesh with proxy configuration:
+
+. From the location where you downloaded your Service Mesh client, type:
++
+`./service-mesh install --clientId=<clientId> --clientSecret=<clientSecret> --license=<license absolute or relative path> --platformUri=<Anypoint Platform Uri> --proxy.host=<host> --proxy.protocol=<protocol> --proxy.port=<port> --proxy.user=<proxy username> --proxy.password=<proxy password>`
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
+| `clientId` | The client ID of the environment (recommended) or the organization. Use the client ID that you obtained as part of the permission and roles prerequisites as explained in this document. | Required | N/A
```

Comment:
```suggestion
| `clientId` | The client ID of the environment (recommended) or the organization. Use the client ID that you obtained as part of the permission and roles prerequisites. | Required | None
```
GLOBALLY replace "N/A" with "none" or "Not applicable." If you must abbr., use "n.a."

Reply:
Done! Thanks!